diff --git a/key_bindings.lua b/key_bindings.lua
index b909107..da66207 100644
--- a/key_bindings.lua
+++ b/key_bindings.lua
@@ -3,6 +3,7 @@ local beautiful = require("beautiful")
 local gears = require("gears")
 local tags = require("tags")
 local panel = require("panel")
+local layouts = require("layouts")
 
 local terminal_emulator = "alacritty"
 
@@ -155,10 +156,20 @@ local global_key_bindings = gears.table.join(
     panel.toggle(awful.screen.focused())
   end),
   awful.key({ "Mod1" }, "m", function()
-    awful.tag.selected().layout = awful.layout.suit.max
-  end),
-  awful.key({ "Mod1" }, "t", function()
-    awful.tag.selected().layout = awful.layout.suit.tile
+    local tag = awful.tag.selected()
+    if tag.layout == awful.layout.suit.tile then
+      tag.layout = awful.layout.suit.max
+    else
+      tag.layout = awful.layout.suit.tile
+    end
+  end),
+  awful.key({ "Mod1" }, "n", function()
+    local tag = awful.tag.selected()
+    if tag.layout == awful.layout.suit.tile then
+      tag.layout = layouts.left_right
+    else
+      tag.layout = awful.layout.suit.tile
+    end
   end)
 )
 
diff --git a/theme.lua b/theme.lua
index 47e71d4..720c839 100644
--- a/theme.lua
+++ b/theme.lua
@@ -1,8 +1,6 @@
 local gears = require("gears")
 local beautiful = require("beautiful")
-
-local theme_name = "dracula"
-local theme = gears.filesystem.get_configuration_dir() .. "themes/" .. theme_name .. "/theme.lua"
+local themes = require("themes")
 
 local function set_wallpaper()
   local wallpaper = os.getenv("HOME") .. "/Pictures/wallpaper.jpg"
@@ -11,7 +9,7 @@ local function set_wallpaper()
 end
 
 local function init()
-  beautiful.init(theme)
+  beautiful.init(themes.dracula)
   set_wallpaper()
 end
 
diff --git a/themes/dracula/theme.lua b/themes/dracula/theme.lua
deleted file mode 100644
index da1a595..0000000
--- a/themes/dracula/theme.lua
+++ /dev/null
@@ -1,33 +0,0 @@
-local beautiful = require("beautiful")
-
-local accent_color = "#614D6B"
-local bg_normal = "#282C34"
-local fg_normal = "#F8F8F2"
-
-local taglist_square_size = beautiful.xresources.apply_dpi(5)
-local taglist_squares_sel = beautiful.theme_assets.taglist_squares_sel(taglist_square_size, fg_normal)
-local taglist_squares_unsel = beautiful.theme_assets.taglist_squares_unsel(taglist_square_size, fg_normal)
-local taglist_squares_urgent = beautiful.theme_assets.taglist_squares_sel(taglist_square_size, bg_normal)
-
-local font = "Iosevka Nerd Font Mono"
-local font_size = 12
-
-return {
-  bg_focus = accent_color,
-  bg_normal = bg_normal,
-  bg_urgent = fg_normal,
-  bg_systray = bg_normal,
-  border_focus = accent_color,
-  border_normal = bg_normal,
-  border_width = 2,
-  dmenu_params = { "-i", "-p", "Run:", "-fn", "'" .. font .. ":" .. font_size .. "'", "-sb", "'" .. accent_color .. "'" },
-  fg_focus = fg_normal,
-  fg_normal = fg_normal,
-  fg_urgent = bg_normal,
-  font = font .. " " .. font_size,
-  panel_height = 23,
-  taglist_squares_sel = taglist_squares_sel,
-  taglist_squares_unsel = taglist_squares_unsel,
-  taglist_squares_urgent = taglist_squares_urgent,
-  useless_gap = 4,
-}
